Dockerfile

```diff
@@ -1,10 +1,16 @@
-FROM rust:slim
+FROM ubuntu:bionic
```

Comment:
Is this just for consistency with the other infra images? I'd rather reuse the `rust` image so this one has fewer commands ...